packages/valory/skills/safe_deployment_abci/rounds.py

```diff
@@ -71,47 +72,47 @@ class RandomnessSafeRound(CollectSameUntilThresholdRound):
     """A round for generating randomness"""
 
     allowed_tx_type = RandomnessPayload.transaction_type
-    payload_attribute = "randomness"
+    payload_attribute = get_name(RandomnessPayload.randomness)
     synchronized_data_class = SynchronizedData
     done_event = Event.DONE
     no_majority_event = Event.NO_MAJORITY
-    collection_key = "participant_to_randomness"
-    selection_key = "most_voted_randomness"
+    collection_key = get_name(SynchronizedData.participant_to_randomness)
```

Comment:
Benefit of RHS over LHS. It's harder to make an error as `mypy` will complain during static checks if the property does not exist. Since we have a 1:1 mapping between properties on `SynchronizedData` and the db keys this then gives us more static checks and it's more easy to follow how things connect.